sources/Debate/src/main/java/com/example/Debate/service/UserServiceImpl.java

```diff
@@ -26,11 +26,25 @@ public UserDto getUserById(String id) {
         return modelMapper.map(user,UserDto.class);
     }
 
+    @Override
+    public UserDto getUserByLogin(String login) {
+        User user = userRepository.findUserByLogin(login).get();
```

Comment:
Possible NullPointerException on get() call - maybe change to orElseThrow(new ResourceNotFoundException(...)).
And for what do we need this method?